src/lib/isDivisibleBy.js

```diff
@@ -1,7 +1,11 @@
 import assertString from './util/assertString';
 import toFloat from './toFloat';
+import isFloat from './isFloat';
 
 export default function isDivisibleBy(str, num) {
   assertString(str);
-  return toFloat(str) % parseInt(num, 10) === 0;
+  if (num.constructor !== Number && !isFloat(num)) {
```

Comment:
```suggestion
  if (!(num instanceof Number) && !isFloat(num)) {
```
`constructor` property is mutable, so is better to check it in this way.

Reply:
nice improvement.

Reply:
used typeof (after a too-quick change on my part) because

```
x = 2
x instanceof Number // false

x = Number(2) // or x = new Number(2)
x instanceof Number // true
```

a primitive number is not a number object. but the idea is still right - using typeof is a better solution.